lib/compiler.js

```diff
@@ -0,0 +1,69 @@
+var vm = require('vm');
+
+function Compiler(options) {
+  this.options = options || {};
+}
+
+module.exports = function(runtime, sourceBundle) {
+  runtime.source = sourceBundle;
+  Compiler.prototype.runtime = runtime;
```

Comment:
So calling this two times will do something strange, right? Perhaps it should return a new `Compiler` constructor each time?